runner/src/run.rs

```diff
@@ -75,6 +75,20 @@ fn get_config(args: &RunArgs) -> Config {
     cfg
 }
 
+fn is_qemu_version_6() -> bool {
```

Comment:
Maybe let's do a slightly more general purpose function `get_qemu_version` which returns the version string.
We might need it for other purposes in the future (e.g. warn that some features might be missing, or simply report the version to the user in case of error, etc...).